### PR TITLE
set unknown content type (but still image) as fallback when unknown

### PR DESCRIPTION
**Pull request by perosb**
workaround for https://github.com/JimBobSquarePants/ImageProcessor/issues/303

### CODE REVIEW

File: src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs [hunk 1]
```diff
@@ -381,11 +381,12 @@ public override void RewritePath(HttpContext context)
                         {
                             HttpResponse contextResponse = context.Response;
                             cachedStream.CopyTo(contextResponse.OutputStream);
+                            string contentType = response.ContentType;
 
                             // Mimetype can be null when returning from the cache.
                             ImageProcessingModule.SetHeaders(
                                 context,
-                                string.IsNullOrWhiteSpace(this.mimeType) ? contextResponse.ContentType : this.mimeType,
```
Comment (JimBobSquarePants) [inline src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs]: There must be a way we can guarantee the correct type? I go to great pains to ensure that the correct type is sent to Azure.

**Comment by perosb** [comment]
@JimBobSquarePants yes, I just updated to PR with the correct fix.

File: src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs (hunk 1 above)
Comment (perosb) [inline src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs]: PR has been updated with correct fix.

Comment (JimBobSquarePants) [inline src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs]: You sure? It still says `"image/xyz"` in the PR.

Comment (perosb) [inline src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs]: @JimBobSquarePants  yes, proper fix is to get ContentType from response/azure.

the "image/xyz" is still there, but is just a fallback if Azure doesn't.

Comment (perosb) [inline src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs]: I mean, the proper fix _is_ included in the PR.

**Pull request merged**